tests/vcr_helper.py

```diff
@@ -1,6 +1,49 @@
+import inspect
 import json
+import os
 import re
 
+current_file_dir = os.path.dirname(os.path.abspath(__file__))
+import vcr
+
+
+def before_record_request(request):
+    filtered_request = filter_hostnames(request)
+    filtered_request_without_dynamic_data = replace_timestamp_in_request(
+        filtered_request
+    )
+    return filtered_request_without_dynamic_data
+
+
+my_vcr = vcr.VCR(
```

Comment:
Put this object instantiation inside the `use_vcr` decorator.  The VCR __init__ is trivial.